app/main.py

```diff
@@ -1,2 +1,94 @@
-class Dictionary:
-    pass
+from typing import Any
+
+
+class Dictionary(object):
```

Comment:
u don't need to inherit Dictionary from object
it's not a Python2, all classes are already inherited from object by default